sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -471,6 +471,13 @@ export class EventHubSender extends LinkEntity {
       options = {};
     }
 
+    const operationTimeoutInMs =
+      options.retryOptions == undefined ||
+        options.retryOptions.operationTimeoutInMs == undefined ||
```

Comment:
Maybe a safer check would be 
```suggestion
        typeof options.retryOptions.operationTimeoutInMs !== "number" ||
```

That way we don't allow it to be set to a different type in the last part of your ternary operation. Could also add an `isFinite(options.retryOptions.operationTimeoutInMs` so we protect against `NaN` or `Infinity` (unless we want Infinity as a valid value).

Reply:
Nice, we should probably add these validations to other such parameters as well